src/components/FarmGroupDetailBody/FarmGroupDetailBody.js

```diff
@@ -18,7 +19,16 @@ export default function FarmGroupDetailBody ({
   // TODO: hide widgets if they have no data, means loading all the data here?
   const widgets = [
     { settings: {}, isVisible: true, name: 'farm_at_a_glance' },
-    { settings: { title: mission, text: group.description || bio }, isVisible: true, name: 'mission' },
+    {
+      name: 'mission',
+      isVisible: true,
+      settings: {
+        title: mission,
+        embeddedVideoURI: group.aboutVideoUri,
+        text: !!group.description && bio,
```

Comment:
this is a little confusing, worth a comment i think saying that if richText is available use that. i am actually not totally sure why we need both fields, cant we just add markdown to the description if it exists and use the one text field?

Reply:
See notes on Slack. This is due to the farm `bio` field. Both options are needed for now.

Reply:
Tibet, as I noted in Slack, we could head this off at the pass by just passing the farm `bio` field into the group description on group creation and then never referencing the bio field again.

Reply:
Hopefully that's what happens and this can be made simpler then. It'll work either way which is nice.